src/main/java/model/Race.java

```diff
@@ -12,32 +12,46 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
 public class Race {
-    private List<Car> cars;
-    private int tryCount;
+    private Cars cars;
 
-    public void setCars (int carCount) {
-        cars = new ArrayList<>(carCount);
-        for (int i = 0; i < carCount; i++) {
-            this.cars.add(new Car());
+    public void createCars (String inputNames) {
+        List originalCars = new ArrayList<>();
```

Comment:
```suggestion
        List<Car> originalCars = new ArrayList<>();
```
http://rangken.github.io/blog/2015/effective-java-4/
무인자 자료형을 사용하기 보단 제네릭을 통해 어떤 클래스의 컬렉션인지 나타내주세요 !